importer/index.js

```diff
@@ -0,0 +1,39 @@
+import csvjson from 'csvjson';
+import fs from 'fs';
+
+class Importer {
+  
+  static async import(filePath) {
+    return await csvJsoAsync(filePath);
+  }
+  
+  static importSync(filePath) {
+    return csvJsonSync(filePath);
+  }
+}
+
+function csvJsoAsync(path) {
+  return new Promise((resolve, reject) => {
```

Comment:
You can use `util.promisify` from node 8, 'pify' or 'es6-promisify' and use async / await with `fs.readFile`.